components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIManagerConfiguration.java

```diff
@@ -139,6 +139,7 @@ public class APIManagerConfiguration {
     private TokenValidationDto tokenValidationDto = new TokenValidationDto();
     private boolean enableAiConfiguration;
     private String hashingAlgorithm = SHA_256;
+    private boolean isEnabled;
```

Comment:
Shall we rename this to a more specific name? Like `isTransactionCounterEnabled`?